src/sentry/services/hybrid_cloud/organization_mapping/model.py

```diff
@@ -19,7 +18,7 @@ class RpcOrganizationMapping(RpcOrganizationSummary):
     date_created: datetime = Field(default_factory=timezone.now)
     verified: bool = False
     customer_id: Optional[str] = None
-    status: Optional[OrganizationStatus] = None
+    status: Optional[int] = None
```

Comment:
👍 
I've secretly been in favor of using raw ints for awhile, but it took some effort to get my PRs in.
Reality is, *nothing technically binds our IntEnums and raw data, and nothing likely should* except for convention of schema evolution.

Reply:
I changed this one as I got trapped in an import loop that would have required moving a ton of imports around to solve.